openedx/core/djangoapps/schedules/content_highlights.py

```diff
@@ -1,3 +1,10 @@
+"""
+Contains methods for accessing weekly course highlights. Weekly highlights is a
+ScheduleExperience built on the Schedules app. The Schedules app introduces
```

Comment:
I'd suggest not capitalizing Schedule and Experience here - since it implies it's a proper noun for something actual.